pkg/kubelet/cm/deviceplugin/device_plugin_handler.go

```diff
@@ -337,19 +344,21 @@ func (h *HandlerImpl) writeCheckpoint() error {
 	if err != nil {
 		return err
 	}
-	filepath := h.devicePluginManager.CheckpointFile()
-	return ioutil.WriteFile(filepath, dataJSON, 0644)
+	err = h.store.Write(defaultCheckpoint, dataJSON)
+	if err != nil {
+		return fmt.Errorf("failed to write checkpoint file %q: %v", defaultCheckpoint, err)
```

Comment:
nit: can we have the error message say `... deviceplugin checkpoint file ...`? Same for the error below.